src/17-infinite-discovery/helpers.ts

```diff
@@ -1,6 +1,7 @@
 import { GravityArtwork } from "./types"
 import path from "path"
 import fs from "fs"
+import _ from "lodash"
```

Comment:
**nitpick:** Not a big deal since this is a script file, and we all have good machines 😅 . Lodash seems to be one of the big dependencies (unpacked Size 1.41 MB). A nicer way is to use only what is needed:  
```suggestion
import { fromPairs, defaults } from "lodash"
```

Reply:
Good call 🙏